packages/app-project/src/shared/components/CollectionsModal/components/SelectCollection/SelectCollection.spec.js

```diff
@@ -48,7 +48,7 @@ describe('Component > SelectCollection', function () {
     await screen.findByRole('listbox')
 
     const searchBar = document.querySelector('input[type="search"]')
-    await user.type(searchBar, 'coll') // causes [TypeError: activeElement.attachEvent is not a function]
+    await user.type(searchBar, ' coll') // causes [TypeError: activeElement.attachEvent is not a function]
```

Comment:
I'm not sure why this changed, but typing 'coll' here sent 'oll' to the Panoptes API. The search input itself seems to be working fine, but the test is skipping the first typed character.

Reply:
Could be that `user.type` isn't waiting for the Grommet text input to be ready, so the first character is typed while the text component isn't focused yet.